### PR TITLE
Adapt tests to new gasless enabled

### PR DESCRIPTION
**Pull request by goncer**
* fix parallel test execution

### CODE REVIEW

**Review by AlexChetverov — APPROVED** [review]

**Review by AlexChetverov — COMMENTED** [review]
Should we use this `getFeeLockMetadata`?

File: test/sequential/feeLock.API.config.test.ts [hunk 1]
```diff
@@ -104,7 +108,7 @@ test("gasless- GIVEN a feeLock WHEN periodLength and feeLockAmount are set THEN
       JSON.stringify(await api?.query.feeLock.feeLockMetadata())
     ).periodLength.toString()
   );
-  const lastFeeLockAmount = new BN(
+  const lastFeeLockAmount = stringToBN(
```
Comment (AlexChetverov) [inline test/sequential/feeLock.API.config.test.ts:111]: Should we use here `getFeeLockMetadata`?

File: test/sequential/feeLock.API.config.test.ts [hunk 2]
```diff
@@ -142,7 +146,7 @@ test("gasless- Changing feeLock config parameter on the fly is works robustly",
   let updateMetadataEvent: any;
   testUser1.addAsset(MGA_ASSET_ID);
 
-  const feeLockAmount = new BN(
+  const feeLockAmount = stringToBN(
```
Comment (AlexChetverov) [inline test/sequential/feeLock.API.config.test.ts:149]: This is the same question about `getFeeLockMetadata`

File: test/sequential/feeLock.API.config.test.ts (hunk 1 above)
Comment (goncer) [reply test/sequential/feeLock.API.config.test.ts:111]: yup, makes sense. will fix !

File: test/sequential/feeLock.API.config.test.ts (hunk 2 above)
Comment (goncer) [reply test/sequential/feeLock.API.config.test.ts:149]: yup, will fix this,

**Pull request merged**